### PR TITLE
Notebook for Creating Cutouts with Astrocut

### PR DESCRIPTION
**Pull request by snbianco**
This notebook checklist has been made available to us by the [the Notebooks For All team](https://github.com/Iota-School/notebooks-for-all/blob/main/resources/event-hackathon/notebook-authoring-checklist.md).
Its purpose is to serve as a guide for both the notebook author and the technical reviewer highlighting critical aspects to consider when striving to develop an accessible and effective notebook.

### The First Cell

- [x] The title of the notebook in a first-level heading (eg. `<h1>` or `# in markdown`).
- [x] A brief description of the notebook.
- [x] A table of contents in an [ordered list](https://www.markdownguide.org/basic-syntax/#ordered-lists) (`1., 2.,` etc. in Markdown).
- [x] The author(s) and affiliation(s) (if relevant).
- [ ] The date first published.
- [x] The date last edited (if relevant).
- [x] A link to the notebook's source(s) (if relevant).

### The Rest of the Cells

- [x] There is only one H1 (`#` in Markdown) used in the notebook.
- [x] The notebook uses other [heading tags](https://www.markdownguide.org/basic-syntax/#headings) in order (meaning it does not skip numbers). 

## Text

- [x] All link text is descriptive. It tells users where they will be taken if they open the link.
- [x] All acronyms are defined at least the first time they are used. 
- [x] Field-specific/specialized terms are used when needed, but not excessively.

## Code

- [x] Code sections are introduced and explained before they appear in the notebook. This can be fulfilled with a heading in a prior Markdown cell, a sentence preceding it, or a code comment in the code section.
- [x] Code has explanatory comments (if relevant). This is most important for long sections of code.
- [x] If the author has control over the syntax highlighting theme in the notebook, that theme has enough color contrast to be legible.
- [x] Code and code explanations focus on one task at a time. Unless comparison is the point of the notebook, only one method for completing the task is described at a time.

### Images

- [x] All images (jpg, png, svgs) have an [image description](https://www.w3.org/WAI/tutorials/images/decision-tree/). This could be
    - [x] [Alt text](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt) (an `alt` property) 
    - [x] [Empty alt text](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#decorative_images) for decorative images/images meant to be skipped (an `alt` attribute with no value)
    - [x] Captions
    - [x] If no other options will work, the image is decribed in surrounding paragraphs.

- [x] Any [text present in images](https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html) exists in a text form outside of the image (this can be alt text, captions, or surrounding text.)

### Visualizations

- [x] All visualizations have an image description. Review the previous section, Images, for more information on how to add it.
- [x] [Visualization descriptions](http://diagramcenter.org/specific-guidelines-e.html) include
    - [x] The type of visualization (like bar chart, scatter plot, etc.)
    - [x] Title
    - [x] Axis labels and range
    - [x] Key or legend
    - [x] An explanation of the visualization's significance to the notebook (like the trend, an outlier in the data, what the author learned from it, etc.)

- [x] All visualizations and their parts have [enough color contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) ([color contrast checker](https://webaim.org/resources/contrastchecker/)) to be legible. Remember that transparent colors have lower contrast than their opaque versions.
- [ ] All visualizations [convey information with more visual cues than color coding](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html). Use text labels, patterns, or icons alongside color to achieve this.
- [ ] All visualizations have an additional way for notebook readers to access the information. Linking to the original data, including a table of the data in the same notebook, or sonifying the plot are all options.

### CODE REVIEW

**Comment by snbianco** [comment]
The Notebook Validation check is failing with this message: `jupyter_client.kernelspec.NoSuchKernel: No such kernel named roman-cal`.

I developed this notebook in the Roman Science Platform, which could be the source of the error. I'm not sure how to resolve it.

**Comment by ttdu** [comment]
@snbianco: you don't need to resolve that error. The CI doesn't test from inside the platform, so it will always give that error.

**Comment by dr-rodriguez** [comment]
A general comment to consider looking into the `warnings` package to see if we can suppress the astropy warnings. We do want to make sure they are not indicative of bigger issues, though, and to address them if needed, but for the notebook should probably not have so many extra warning messages.

**Comment by dr-rodriguez** [comment]
Also, it would be good to have both the fits and asdf cutouts be the same area to demonstrate that nothing is different between the two.

**Comment by snbianco** [comment]
Just pushed changes to suppress the warnings and change the parameters for the ASDF cutout to match the FITS cutout.

File: notebooks/roman_cutouts/roman_cutouts.ipynb [hunk 1]
```diff
@@ -0,0 +1,806 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "69ac82b9-57ab-4b7c-854d-a5e74698d1ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_cutout_pixel(img):\n",
+    "    plt.figure(figsize=(6, 6))\n",
```
Comment (havok2063) [inline notebooks/roman_cutouts/roman_cutouts.ipynb:430]: can you add a single-line docstring here with a small text of what this helper function does?

File: notebooks/roman_cutouts/roman_cutouts.ipynb [hunk 2]
```diff
@@ -0,0 +1,806 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "337ee97b-19ce-4df3-9963-50d64723aafb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_cutout_world(cutout, img, coord, wcs):        \n",
+    "    # plot the figure with wcs\n",
```
Comment (havok2063) [inline notebooks/roman_cutouts/roman_cutouts.ipynb:455]: Same thing here.  Add a single-line docstring with some small text of what this helper function does.

**Review by havok2063 — COMMENTED** [review]
Overall this looks good to me.

**Comment by dr-rodriguez** [comment]
I just realized this hasn't been merged yet but is part of deliveries for Roman B15. 
Is there any reason why we shouldn't merge at this point?

**Review by dr-rodriguez — APPROVED** [review]

**Comment by snbianco** [comment]
@cmurray-astro let us know that the Roman mission office wants to wait until after the next round of tests happening in early October. We will have to check out the notebook on ops RSP for the demos.

**Comment by dr-rodriguez** [comment]
I'm not sure if we can wait til October- this notebook is a deliverable for B15 which is due September. I'll include Claire in the email chain to get some clarity.

**Comment by scfleming** [comment]
There is nothing stopping us from, e.g., going into RSP and checking out the notebook in the user's local environment for running the demo. This conversation was about when the notebook would appear automatically for ALL users in RSP.  That shouldn't impact the ability to create test instructions, or run the demo, I would think.

**Comment by scfleming** [comment]
Since it's demo-only, we could start off the demo with the notebook already available in the user's RSP working directory.

**Comment by ttdu** [comment]
@tddesjardins likely has better insights than Claire about RMO's motivation for not merging. As I understand it, Sam is correct; since the astrocut notebook wasn't a part of the original RSP plan, they're hesitant to add it during the testing phase.

**Pull request merged**